SS14.Launcher/HappyEyeballsHttp.cs

```diff
@@ -19,7 +20,8 @@ public static HttpClient CreateHttpClient()
         var handler = new SocketsHttpHandler
         {
             ConnectCallback = OnConnect,
-            AutomaticDecompression = DecompressionMethods.All
+            AutomaticDecompression = DecompressionMethods.All,
+            ConnectTimeout = new TimeSpan(0, 0, 10),
```

Comment:
It's probably better to set timeouts locally via `CancellationToken`. We do this for the server status fetching logic.